gendst/data/data.go

```diff
@@ -1961,7 +1982,8 @@ var Info = map[string][]Part{
 			PositionField: Field{"Assign"},
 		},
 		Decoration{
-			Name: "Name",
+			Name: "AliasName",
+			Use:  Expr(func(n *jen.Statement) *jen.Statement { return n.Dot("Assign").Dot("IsValid").Call() }),
```

Comment:
Was adding `Use` intentional? I'm hazy on how this works, but it doesn't feel like this was needed. As before renaming `Name` will break backwards compatibility, so I assume it was a mistake?

Reply:
See above.